conf/config.json

```diff
@@ -1,5 +1,10 @@
 {
-    "default_server_name": "__DEFAULT_HOME_SERVER__",
+      "default_server_config": {
+        "m.homeserver": { "base_url": "__DEFAULT_HOME_SERVER__" },
+        "im.vector.riot.jitsi": {"preferredDomain": "jitsi.riot.im"},
+        "im.vector.riot.e2ee": {"default": true },
```

Comment:
For the jitsi server URL and e2e parameters we already have a setting in the synapse server. So if we define this in both side which setting override the other one ? Can we specify to elements to use the server paramer ?

Reply:
It seems sysnapse config goes first. 
Maybe for confusion its better to remove this 2 lines from here.